### DISCUSSION TITLE
Change the Penguin emoji to Tux

### DISCUSSION THREAD

**Discussion by Secret-chest** [C1710/blobmoji#143] [Ideas]
because why not

**Comment by ikkqqstudio** [comment]
@DavidBerdik Are You Going To Add This? I use Debian GNU/Linux KDE With Blob Emojis

**Reply by DavidBerdik** [reply]
I think it would be great to use Tux for the penguin emoji! My version of the blob emoji set [already uses Clippy](https://github.com/DavidBerdik/blobmoji2/blob/blobmoji-master/svg/emoji_u1f4ce.svg) for the paperclip!

If somebody wants to make a design that can be identified as Tux while also adhering to the style used by the rest of the emojis in the set, I am willing to do this.

**Reply by ikkqqstudio** [reply]
who the frick disliked my comment

**Reply by ikkqqstudio** [reply]
> My version of the blob emoji set [already uses Clippy](https://github.com/DavidBerdik/blobmoji2/blob/blobmoji-master/svg/emoji_u1f4ce.svg) for the paperclip!

Okay, I Already Get It
